.markdownlint.json

```diff
@@ -0,0 +1,38 @@
+{
+    "_docs_for_this_file": "https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md",
+    "line-length": false,
+    "first-line-h1": true,
+    "first-header-h1": {
+        "level": 1
+    },
+    "header-style": {
+        "style": "atx"
+    },
+    "ul-style": {
+        "style": "asterisk"
+    },
+    "hr-style": {
```

Comment:
i see line breaks used to infrequently that i'd just leave this out and go with the default "consistent"